addons/stock_landed_costs/models/stock_move.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+class StockMove(models.Model):
```

Comment:
```suggestion


class StockMove(models.Model):
```
I can't find a bigger nitpicking haha
(pep8 e302)